lib/std/os/uefi.zig

```diff
@@ -51,7 +93,7 @@ pub const IpAddress = extern union {
 
 /// GUIDs are align(8) unless otherwise specified.
 pub const Guid = extern struct {
-    time_low: u32,
+    time_low: u32 align(8),
```

Comment:
Please add a comptime assert that ensures the natural alignment of `Guid` is actually 8 now that we've dropped explicit alignment specifiers in most places